src/backend/Dockerfile.rhel8

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM registry.access.redhat.com/ubi8/dotnet-60-runtime AS base
+WORKDIR /opt/app-root/app
+EXPOSE 8080
+ENV ASPNETCORE_URLS=http://*:8080
```

Comment:
You should not set this.  See https://github.com/redhat-developer/s2i-dotnetcore/tree/master/6.0/runtime#environment-variables

Reply:
Thanks! Updated.